lib/engine/game/g_1848/step/tracker.rb

```diff
@@ -9,10 +9,7 @@ module Tracker
         include Engine::Step::Tracker
 
         def lay_tile_action(action)
-          # Yellow cities on K tiles need the K label added
-          add_k = action.hex.tile.label.to_s == 'K'
           super
-          action.hex.tile.label = 'K' if add_k
 
           was_connected = @game.sydney_adelaide_connected
           now_connected = @game.check_sydney_adelaide_connected
```

Comment:
You do not want to check graph on every tile lay. This is a HUGE performance hit. Take a look at the CheckNYCFormation  step for an example of how to use auto actions.

Reply:
To be fixed in next PR